monitoring/base/rhods-rules.yaml

```diff
@@ -16,5 +16,9 @@ spec:
       expr: count(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}==1)
     - record: cluster:usage:consumption:rhods:active_users
       expr: count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}[1h]))
+    - record: cluster:usage:consumption:cpu:requests:runtime
+        expr: sum(kube_pod_container_resource_requests{namespace='rhods-notebooks', resource='cpu'})
+    - record: cluster:usage:consumption:cpu:limits:runtime
```

Comment:
@DharmitD same thing here about this metric name needing to have "rhods" in it somewhere. I'd recommend:
`cluster:usage:consumption:rhods:cpu_limits_runtime`

Reply:
Done, updated the name to have "rhods".